<launch>
    <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_world.launch"/>
    <node name="object_detection_node" pkg="turtlebot-perception-navigation" type="object_detection_node.py" output="screen">
        <param name="yolo_model_path" value="$(find turtlebot-perception-navigation)/yolo/yolo11n.pt"/>
        <param name="confidence_threshold" value="0.25" />
    </node>
    <node name="movement_node" pkg="turtlebot-perception-navigation" type="movement_node.py" output="screen"/>
    <node name="voice_command_node" pkg="turtlebot-perception-navigation" type="voice_command_node.py" output="screen"/>
</launch>